content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -3171,6 +3171,8 @@ Represents **default values** that cannot be expressed in the Prisma schema (suc
   - [Set default values for `Unsupported` types](#set-default-value-for-unsupported-type)
   - [Override default value behavior for supported types](#override-default-value-behavior-for-supported-types)
 
+- String values in `dbgenerated` may not match what the DB returns as the default value, because values such as strings may be explicitly cast (e.g. `'hello'::STRING`). When a mismatch is present, Prisma Migrate will indicate a migration is still needed. You can use `prisma db pull` to infer the correct value to resolve the discrepancy. ([Related issue](https://github.com/prisma/prisma/issues/14917))
```

Comment:
```suggestion
- String values in `dbgenerated` might not match what the DB returns as the default value, because values such as strings may be explicitly cast (e.g. `'hello'::STRING`). When a mismatch is present, Prisma Migrate indicates a migration is still needed. You can use `prisma db pull` to infer the correct value to resolve the discrepancy. ([Related issue](https://github.com/prisma/prisma/issues/14917))
```

Thanks for this contribution to our docs. We appreciate it!

I suggest a small wording change suggestion, to clarify this sentence ("may not" can be read as "are not permitted to, but I think the meaning here is "might not").

I also made a suggestion to the tense, as per our style guide.